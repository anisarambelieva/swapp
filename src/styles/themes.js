const themes = {
  light: {
    background: "#FFF",
    header: "#000",
    text: {
      primary: "#4BD5EE",
      secondary: "#4E5B6E",
      description: " #4E5B6E",
    },
    button: {
      primary: {
        bg: "#000",
        text: "#FFE300",
      },
      secondary: {
        bg: "#EFF2F7",
        text: "#4BD5EE",
      },
    },
  },
  dark: {
    background: "#333",
    header: "#333",
    text: {
      primary: "#FFE300",
      secondary: "#4BD5EE",
      description: " #ABB1BA",
    },
    button: {
      primary: {
        bg: "#4BD5EE",
        text: "#FFE300",
      },
      secondary: {
        bg: "#3C4858",
        text: "#4BD5EE",
      },
    },
  },
};

export default themes;
