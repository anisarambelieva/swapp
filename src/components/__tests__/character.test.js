import { render } from "@testing-library/react";
import { ThemeProvider } from "styled-components";
import themes from "../../styles/themes.js";

import Character from "../character.js";

describe("<Character />", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <ThemeProvider theme={themes["light"]}>
        <Character imageSrc="image" name="Star wars: Episode 1" />
      </ThemeProvider>
    );

    expect(container).toMatchSnapshot();
  });
});
