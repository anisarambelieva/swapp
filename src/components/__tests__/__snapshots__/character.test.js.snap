// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Character /> should match snapshot 1`] = `
.c0 {
  background-color: white;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 20px;
  font-weight: bold;
  color: #4bd5ee;
  height: 100%;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 0px 5px 5px 0px;
}

<div>
  <div
    class="row"
  >
    <div
      class="col-md-4"
      style="padding-right: 0px;"
    >
      <img
        class=""
        src="image"
        style="height: 100px; object-fit: cover; width: 100%; border-radius: 5px 0px 0px 5px;"
      />
    </div>
    <div
      class="col"
      style="height: 100px; padding-left: 0px;"
    >
      <h1
        class="c0"
      >
        Star wars: Episode 1
      </h1>
    </div>
  </div>
</div>
`;
